docs/source/tutorials.rst

```diff
@@ -451,44 +451,10 @@ BTK supports galaxy image generation with galsim_hub ; please refer to :ref:`thi
 
 You can find a notebook version of this tutorial in the notebooks folder.
 
-First, you should use the ``CosmosCatalog`` catalog instead of the Catsim one. While galsim_hub only require parameters for the image generation, we have chosen to use COSMOS as the source of those parameters so as to get a realistic distribution of those parameters. We have included a small sample of the catalog in BTK, and advise you to download the full catalog (see:ref:`COSMOS`) for better results.
+.. Add notes on how to install galsim hub with pip extras
 
-.. jupyter-execute::
-
-  COSMOS_CATALOG_PATHS = [
-    "../data/cosmos/real_galaxy_catalog_23.5_example.fits",
-    "../data/cosmos/real_galaxy_catalog_23.5_example_fits.fits",
-  ]
-  catalog = btk.catalog.CosmosCatalog.from_file(COSMOS_CATALOG_PATHS)
-
-We then instantiate the sampling function ; you should use the one specific for galsim_hub, which includes a cut on the size of the galaxies, as artifacts tend to appear when trying to generate galaxies which are too big.
 
-.. jupyter-execute::
+Advanced features
+------------------
 
-  stamp_size = 24.0
-  sampling_function = btk.sampling_functions.DefaultSamplingGalsimHub(stamp_size=stamp_size)
-
-Then we can instantiate the ``DrawBlendsGenerator`` with the survey of your choice. Please bear in mind that while BTK will draw the images in any band you desire, galsim_hub does not generate a SED for the galaxy ; this means that the magnitude will be inacurrate in any other band than the one generated by the galsim_hub model you use (by default ``"hub:Lanusse2020"``).
-
-.. .. jupyter-execute::
-
-..   draw_generator = btk.draw_blends.GalsimHubGenerator(
-..       catalog,
-..       sampling_function,
-..       [Rubin],
-..       batch_size=8,
-..       stamp_size=stamp_size,
-..       shifts=None,
-..       indexes=None,
-..       cpus=1,
-..       add_noise=True,
-..       galsim_hub_model="hub:Lanusse2020", #May be replaced by any model compatible with galsim_hub
-..       param_names=["flux_radius", "mag_auto", "zphot"], #Name of the parameters ; they must match with the model you provide
-..   )
-
-.. jupyter-execute::
-
-  batch = next(draw_generator)
-  blend_images = batch['blend_images']
-  blend_list = batch['blend_list']
-  btk.plot_utils.plot_blends(blend_images, blend_list, limits=(30,90))
+You can find a detailed tutorial on several more advanced feature of BTK in this `notebook <https://github.com/LSSTDESC/BlendingToolKit/blob/main/notebooks/intro.ipynb>`_, including how to write your own measure function which should be interesting to most user. You will also see how to use the multiresolution, write your own sampling function or survey, use several measure functions and use of the measure_kwargs feature.
```

Comment:
Did you mean to reference the `advanced-features` notebook here?

Reply:
Yes, I forgot to change it.